client/src/main/java/org/apache/rocketmq/client/producer/DefaultMQProducer.java

```diff
@@ -133,6 +133,11 @@ public class DefaultMQProducer extends ClientConfig implements MQProducer {
      */
     private TraceDispatcher traceDispatcher = null;
 
+    /**
+     * Indicate whether to block message when asynchronous sending traffic is too heavy.
+     */
+    private boolean enableBackpressureForAsyncMode = true;
+
```

Comment:
By default, this argument should be false to behave as the old client.

And the same time, we could suggest setting enableBackpressureForAsyncMode to true in the example code.

Reply:
ok, I have fix it.